.github/workflows/integrate.yaml

```diff
@@ -55,8 +55,9 @@ jobs:
         uses: charmed-kubernetes/actions-operator@main
         with:
           provider: microk8s
-          channel: 1.25-strict/stable
+          channel: 1.26-strict/stable
```

Comment:
Ideally we'd have this change in a different PR. Also, we may want to bump this to 1.29, but first I'd like to understand if this change was made because it is not working with 1.25.

Reply:
I just confirmed with @misohu that this change is required for kserve 0.13 to work. No further action is required.